### ISSUE TITLE
Systematically prevent hyphenation before combining characters

### ISSUE THREAD

**Issue by reutenauer** [hyphenation/tex-hyphen#5]
To be discussed: add all combining characters as non-hyphenatable-before, i. e.

    8̀
    8́
    8̂

etc.

See http://tug.org/pipermail/tex-hyphen/2016-November/001589.html and subsequent discussion.

**Comment by Davislor** [comment]
I reently encountered this bug [in a project in ancient Greek.](https://tex.stackexchange.com/questions/571643/why-does-hyphenation-orphan-combining-accents). A similar bug was sighted [for Russian in 2017.](https://tex.stackexchange.com/questions/339242/avoid-hyphenation-before-combining-accents)

I strongly agree. Inserting a hyphenation point before combining characters violates the Unicode standard, and even if there’s some bizarre corner case where it makes sense, the default should very definitely be not to hyphenate before any Unicode combining character.

**Comment by reutenauer** [comment]
I strongly agree with myself too :-)  But I wouldn’t hold my breath as I’m currently the only one doing all of the work on TeX-hyphen ...
